### PR TITLE
[Doc] Update instructions for 7.0 upgrade

### PR DESCRIPTION
**Pull request by karenzone**
Updated instructions for 7.0 upgrade. Added format/info on PQ upgrade info pre- and post-6.3.
I have not tested/verified these instructions at this time. This PR is to get the framework and versions in place.

### CODE REVIEW

**Comment by karenzone** [comment]
@lcawl  Tagging you as requested.

File: docs/static/upgrading.asciidoc [hunk 1]
```diff
@@ -66,37 +124,42 @@ Keep in mind that only one Logstash instance can write to `path.queue`. You
...
-in earlier versions. For example, if you attempt to send the 5.x template to a cluster before Elasticsearch 5.0, then it
-will not be able to use it and all indexing will fail likely fail. If you use your own, custom template with Logstash,
+Although we make great efforts to ensure compatibility, Logstash 7.0 is not completely backwards compatible. As noted
+in the Elastic Stack upgrade guide, Logstash 7.0 should not be upgraded before Elasticsearch 7.0. This is both
+practical and because some Logstash 7.0 plugins may attempt to use features of Elasticsearch 7.0 that did not exist
+in earlier versions. For example, if you attempt to send the 7.x template to a cluster before Elasticsearch 7.0, then 
+all indexing will fail likely fail. If you use your own custom template with Logstash,
 then this issue can be ignored.
 
 Note the Elasticsearch Output Index Template change in the <<breaking-changes>> documentation for further insight into
```
Comment (karenzone) [inline docs/static/upgrading.asciidoc]: At line 165, does the note about Output Index Template breaking change come out for 7.0?

**Comment by karenzone** [comment]
@guyboertje or @jsvd  Can you take a look, please? We'll likely need to make more changes, but I'd like to get this merged for stack upgrade testing.

File: docs/static/upgrading.asciidoc (hunk 1 above)
Comment (guyboertje) [reply docs/static/upgrading.asciidoc]: I suppose, if the user upgraded via 6.7 then they might have read this Note

**Comment by karenzone** [comment]
@guyboertje   Thank you!  Iteration continues starting now.

**Pull request closed without merging**